src/Gui/Inventor/Draggers/SoLinearDragger.h

```diff
@@ -47,18 +51,14 @@ namespace Gui
 class SoLinearDragger : public SoDragger
 {
     SO_KIT_HEADER(SoLinearDragger);
-    SO_KIT_CATALOG_ENTRY_HEADER(activeSwitch);
-    SO_KIT_CATALOG_ENTRY_HEADER(activeColor);
     SO_KIT_CATALOG_ENTRY_HEADER(translator);
+    SO_KIT_CATALOG_ENTRY_HEADER(activeSwitch);
+    SO_KIT_CATALOG_ENTRY_HEADER(secondaryColor);
```

Comment:
what does `secondary` mean in that context?

Reply:
Basically, the SoBaseColor that is active when you are dragging. activeColor is already taken so I used that. Can you suggest a better name?

Reply:
How it is different from `activeColor`?

Reply:
activeColor is an SoSFColor (a field) and secondaryColor is a SoBaseColor (a node). The field is connected with the node's rgb property.

Reply:
Is this (^) explanation enough? Or something else needs to be done for this one.